service/controller/internal/cloudconfig/tccpn.go

```diff
@@ -0,0 +1,98 @@
+package cloudconfig
+
+import (
+	"context"
+
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/aws-operator/service/controller/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/key"
+	"github.com/giantswarm/certs"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_5_0_0"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/randomkeys"
+)
+
+type TCCPNConfig struct {
+	Config Config
+}
+
+type TCCPN struct {
+	config Config
+}
+
+func NewTCCPN(config TCCPNConfig) (*TCCPN, error) {
+	err := config.Config.Validate()
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	t := &TCCPN{
+		config: config.Config,
+	}
+
+	return t, nil
+}
+
+//TODO LH-AVDD figure out what should go in render from copy of tccp
```

Comment:
I see you may want to address the TODOs in separate PRs. My thinking with the Cloud Config rendering was that this should simply be as it was with the single master approach. So the code commented below should be activated and "just work". What was missing here?